packages/cli/src/ceramic-daemon.ts

```diff
@@ -336,6 +337,7 @@ export class CeramicDaemon {
     baseRouter.use('/collection', collectionRouter)
     baseRouter.use('/admin/getCode', adminCodesRouter)
     baseRouter.use('/admin/models', adminModelRouter)
+    baseRouter.use('/admin/nodeStatus', adminNodeStatusRouter)
```

Comment:
I thought about putting this under the existing `node` router so the endpoint would be `node/status` instead of `admin/nodeStatus`.  But right now all the functionality that is gated by a JWS from an admin DID is under the `admin` router so I decided to keep with that pattern for now.  Don't feel strongly though and would be happy to move to `/node/status` if that seems better.

@oed @PaulLeCam

Reply:
How about `/admin/status`?

Reply:
I'll do in the next PR to avoid merge conflicts